main/guides/getting-started/index.md

```diff
@@ -74,6 +102,13 @@ yarn --version # for verification
 
 ### Installing Docker
 
+<details>
+<summary>Installing Docker</summary>
+
+For docker, you can either download Docker UI using the steps on the [installation for Ubuntu](https://docs.docker.com/desktop/install/ubuntu/) or [installation for Ubuntu](https://docs.docker.com/desktop/install/mac-install/)
```

Comment:
the second link should be 'installation for Mac'. OR you can simply provide link for both at https://docs.docker.com/get-docker/